.github/workflows/test.yml

```diff
@@ -48,38 +44,31 @@ jobs:
       CKAN_REDIS_URL: redis://redis:6379/1
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Install requirements
+    - uses: actions/checkout@v3
+    - name: Create solr container
       run: |
-        pip install -r requirements.txt
-        pip install -r dev-requirements.txt
-        pip install -e .
-        pip install -U requests[security]
-    - name: Install required ckanexts
+        /usr/bin/docker create --name test_solr --network ${{ job.container.network }} --network-alias solr \
+          --workdir /__w/ckanext-stadtzh-harvest/ckanext-stadtzh-harvest --publish 8983:8983 \
+          -e "SOLR_HEAP=1024m" -e "SOLR_SCHEMA_FILE=/opt/solr/server/solr/configsets/ckan/conf/managed-schema" \
```

Comment:
Is this where it is not advisable/working to use a config-variable for e.g. `/opt/solr/server/solr/configsets/ckan/conf/` ?

Reply:
Hmm, actually, it would probably work OK here. The problem was coming when trying to create the solr container as a `service`. I can try again and see if it works now - it would definitely make this easier to read. :)

Reply:
Fixed, thank you for the hint :)